src/event_model/basemodels/__init__.py

```diff
@@ -16,20 +16,20 @@
 from event_model.basemodels.stream_datum import StreamDatum
 from event_model.basemodels.stream_resource import StreamResource
 
-DocumentType = Union[
-    Type[Datum],
-    Type[DatumPage],
-    Type[Event],
-    Type[EventDescriptor],
-    Type[EventPage],
-    Type[Resource],
-    Type[RunStart],
-    Type[RunStop],
-    Type[StreamDatum],
-    Type[StreamResource],
+DocumentType = Union[  # noqa: UP007
```

Comment:
I figured it would be easier to just tell ruff to shut up than to reformat everything in a single line...

Reply:
Ah, nevermind:

```py
MyInt = (
    type[int] |
    type[str] |
    type[bytes]
)

this seems to be a valid syntax and it is interpreted correctly as the union of multiple types. I'll apply changes.